plugins/lookup/onepassword.py

```diff
@@ -502,7 +518,9 @@ def assert_logged_in(self):
             # an interactive prompt. Only run 'op account get' after first listing accounts to see
             # if there are any previously configured accounts.
             args = ["account", "get"]
-            if self.subdomain:
+            if self.account_id:
+              args.extend(["--account", self.account_id])
```

Comment:
Please note the sanity test failures, among them:
```suggestion
                args.extend(["--account", self.account_id])
```